test/wpt/test-url.js

```diff
@@ -5,4 +5,5 @@ const { WPTRunner } = require('../common/wpt');
 
 const runner = new WPTRunner('url');
 
+runner.setInitScript('global.Window = global.__proto__.constructor;');
```

Comment:
```suggestion
runner.setInitScript('global.Window = Object.getPrototypeOf(global).constructor;');
```

btw why is this needed?

Reply:
idlharness.any.js will check the environment. e.g. Window, ServiceWorkerGlobalScope, etc. We should match at least one of them or the test cases will failed.

Reply:
Interesting. This trick may be useful for other wpt too 👍🏻

Reply:
Refs: https://github.com/nodejs/node/blob/24673ace8ae196bd1c6d4676507d6e8c94cf0b90/test/fixtures/wpt/resources/idlharness.js#L654

Reply:
I think I can make it as a new util function.

Reply:
@targos I've created a new method named `pretendGlobalThisAs('foo')`.